shared/editor/nodes/OrderedList.ts

```diff
@@ -22,6 +22,10 @@ export default class OrderedList extends Node {
           default: 1,
           validate: "number",
         },
+        listStyle: {
+          default: "number",
+          validate: "string",
```

Comment:
Lets make this a function that checks for one of the three valid values.

([docs](https://prosemirror.net/docs/ref/#model.AttributeSpec.validate))